junit-platform-launcher/src/main/java/org/junit/platform/launcher/TestIdentifier.java

```diff
@@ -50,10 +42,6 @@ public final class TestIdentifier implements Serializable {
 
 	@Serial
 	private static final long serialVersionUID = 1L;
```

Comment:
This should be incremented to indicate to Java that the pre-6.0 version of this class and this version are not compatible regarding serialization.

```suggestion
	private static final long serialVersionUID = 2L;
```